lib/signatures.ts

```diff
@@ -1,25 +1,34 @@
 import { promises } from 'fs';
-import { SignatureHashAlgorithm, SignatureResult } from './types';
+import { SignatureFormat, SignatureResult } from './types';
```

Comment:
I would suggest to keep the type naming as it was `SignaturesHashAlgorithm` rather than
`SignatureFormat`, as it does not fit the domain that we are dealing with `hash algorithms`

Reply:
`SignatureFormat` was actually my suggestion to @dekelund, so I can take the blame for that. I don't have a strong opinion on it, but internally in the Shinobi team, we refer to these as "signature formats". Although signature formats are related to hash algorithms, I would prefer that we reserve that term for the underlying hash algorithm, e.g. md5. In the future, signature formats may make use of one or more hash algorithms.

Reply:
I understand your point @robguinness-snyk and it made sense for me when I saw it the first time, albeit I think assign uHash rather than e.g format number 1 or 2 could be problematic later on, I would prefer us to keep this as simple as possible in this early stage, as this model will change.

Just thought here, what if for any reason we desire of use uHash also for snippets?
We would end up with another format right? 

What do you think if we have a chat about this when I am back? 

I think there’s a good opportunity to discuss this model and define good things.

Reply:
I have no big opinion about this, it seems reasonable to use the same term between teams, but if there exists good reason not to, then lets stick with the previous name.

Reply:
Hey @dekelund, @robguinness-snyk and me had a quick chat, and we decided for now to stick with the previous name.

Reply:
👍